online/src/test/scala/ai/chronon/online/test/CatalystUtilTest.scala

```diff
@@ -584,4 +584,30 @@ class CatalystUtilTest extends TestCase with CatalystUtilTestSparkSQLStructs {
     assertTrue(res.get("c").asInstanceOf[util.ArrayList[Any]].contains("world"))
   }
 
+  def testWhereClauseShouldFilterEventOut(): Unit = {
```

Comment:
@caiocamatta-stripe - should we add a test that covers the case where we include the row and confirm the projected columns etc match what we expect?